### PR TITLE
fix: redirect hubs.mozilla.com to sumo article

### PR DESCRIPTION
**Pull request by pattersonbl2**
## Refractr PR Checklist

JIRA ticket: https://mozilla-hub.atlassian.net/browse/SVCSE-1946

When creating a PR for Refractr, confirm you've done the following steps for smooth CI and CD experiences:
- [ ] Have you updated the relevant YAML in the PR?
- [ ] Have you checked the relevant YAML for any possible dupes regarding your domain?
- [ ] Have you checked if there are any TLS cert concerns - e.g. if the domain being redirected already exists, and it is being changed to point at Refractr, is a temporary TLS 'outage' while waiting for Lets Encrypt certification via HTTP challenge okay? If not, [have you followed these steps for using DNS challenges with our cert-manager setup](https://mana.mozilla.org/wiki/display/SRE/Refractr+-+How+To+-+DNS+Challenges)?
- [ ] If desired, have you generated the Nginx manually to confirm addition works as expected? 
- [ ] If desired, are you able to connect to EKS (cluster itse-apps-prod-1, namespace fluxcd) to more closely monitor the deploys?

After PR merge, next steps include:
- [ ] If going straight from main merge & Stage deploy to a release & production deploy, create the relevant GitHub release with an incremented version / tag applied.
- [ ] Confirm you are ready and able to perform the requested DNS creation or change post-deploy?

### CODE REVIEW

**Comment by pattersonbl2** [comment]
For context,  The hubs service in GCP will be shutdown and the project deleted.   I am not sure if this changes the process.

**Comment by gcoxmoz** [comment]
Note that there are other hubs URLs in the refractr config that you may want to consolidate/clean as a part of this winddown.  I don't /think/ this is a blocker, but pointing it out while it's fresh.

File: prod-refractr.yml [hunk 1]
```diff
@@ -344,6 +344,12 @@ refracts:
   # SE-1564
 - support.mozilla.org/en-US/kb/canary-domain-use-application-dnsnet:  use-application-dns.net
 
+  # SVCSE-1946
+- support.mozilla.org/en-US/kb/end-support-mozilla-hubs:
+    - hubs.mozilla.org
+    - hubs.mozilla.com
+    - hubs.mozilla.com/labs/
```
Comment (bkochendorfer) [inline prod-refractr.yml:351]: I think you can remove the path here if you are redirecting the entire domain it would never get hit

```suggestion
```

**Review by bkochendorfer — CHANGES_REQUESTED** [review]

**Comment by pattersonbl2** [comment]
@gcoxmoz @bkochendorfer  I have pushed a change to clean up the redirect url's and removed the  /lab url.

**Comment by gcoxmoz** [comment]
I have no review power here, but conceptual f+

**Review by bkochendorfer — APPROVED** [review]
LGTM, let me know when you want to us to deploy.

**Comment by pattersonbl2** [comment]
@bkochendorfer  They are planning to start the shutdown around 11am est time.  I want to give time for the main gke cluster to be shutdown so i am thinking 1 pm est.

**Pull request merged**